src/pages/default/connections/ConnectionDetail.tsx

```diff
@@ -6,29 +6,21 @@ import { formatDID } from "@/util/formatters";
 import { Layouts } from "@/theme/layouts";
 import { Typography } from "@/theme/typography";
 import { Tappable } from "../Tappable";
-import { BadgeNames, ItemProps } from "@/components/Item";
+import { BadgeNames } from "@/components/Item";
 import { mockConnections } from "@/services/mocks";
 import { profilesAtom } from "@/features/identity/atoms";
+import { For } from "@legendapp/state/react";
+
+const tabLabels = ["About", "Connections", "Activity"];
 
 const ConnectionDetailScreen = ({ navigation, route }) => {
   const { heading: connectionName, iconName: icon } = route.params.connection;
-  const tabLabels = ["About", "Connections", "Activity"];
   const [activeTab, setActiveTab] = useState(tabLabels[0]);
 
-  const navigateToReviewConnection = (connectionPair) => {
-    navigation.navigate("ReviewConnection", { connectionPair });
```

Comment:
this isn't how we route in RN (and is basic react prop drilling). i'll plan to introduce routing when there's something to route to and a store to route against.